model/edx.go

```diff
@@ -67,6 +67,29 @@ func (e *EdxLMS) GetOAuthConsumerSecret() string {
 	return e.OAuthConsumerSecret
 }
 
+func (e *EdxLMS) GetTeams() map[string]string {
+	return e.Teams
+}
+
+func (e *EdxLMS) GetLearningGroupPrefixes() []*LearningGroupPrefix {
+	// Instantiate array of pointers to LearningGroupPrefix structs
+	var size int = len(e.PersonalChannels.ChannelList)
+	var prefixes []*LearningGroupPrefix = make([]*LearningGroupPrefix, size)
+
+	// For every key in PersonalChannels.ChannelList in the active configuration
+	counter := 0
+	for personalChannelName, _ := range e.PersonalChannels.ChannelList {
+		// Add the pointer to this LearningGroupPrefix struct to the array of pointers
+		prefixes[counter] = &LearningGroupPrefix{Prefix:personalChannelName}
+
+		// Increment counter in loop (using range over key, value types doesn't produce a counter)
```

Comment:
Also good call. I think I added the comment because my initial thought looking at this would be, 'doesn't a for loop come with a counter usually?'. But with a more descriptive variable name for the counter, I think this comment is safe to remove.